deployment/network-operator/charts/sriov-network-operator/values.yaml

```diff
@@ -42,7 +42,7 @@ resources: {}
 
 # Image URIs for sriov-network-operator components
 images:
-  operator: mellanox/sriov-operator:7ad0f8e
+  operator: mellanox/sriov-operator:7ad0f8e-k8s
```

Comment:
maybe <commit>-rel1

Reply:
I don't have strong opinion on it. We can push new tag for the image. I added 'k8s' suffix to make it self-descriptive.